lib/ansible/modules/cloud/misc/terraform.py

```diff
@@ -104,6 +104,12 @@
       - A group of key-values to provide at init stage to the -backend-config parameter.
     required: false
     version_added: 2.7
+  destroy:
+    description:
+      - Create a destroy plan, used when C(state) is C(planned) 
+    required: false
+    version_added: 2.9
```

Comment:
```suggestion
    version_added: 2.10
```

Reply:
I think this needs to be `version_added: "2.10"` to prevent YAML parsing this into a float and dropping the trailing zero.